packages/webchat/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@botpress/webchat",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "license": "AGPL-3.0",
+  "scripts": {
+    "build": "yarn --silent && tsc --build",
+    "watch": "yarn --silent && tsc --build --watch"
+  },
+  "files": [
+    "dist"
+  ],
+  "devDependencies": {},
+  "dependencies": {
+    "@botpress/messaging-socket": "0.0.1",
+    "mobx-react": "^5.4.4"
+  }
+}
```

Comment:
Another thing that isn't copy pasted here is this pretty basic package.json. We start with a dependency on the socket package.